src/libraries/Microsoft.PowerFx.Connectors/Tabular/CdpTableResolver.cs

```diff
@@ -24,10 +24,21 @@ internal class CdpTableResolver : ICdpTableResolver
 
         private readonly HttpClient _httpClient;
 
-        private readonly string _uriPrefix;
+        [Obsolete]
+        private readonly string _uriPrefix = null;
```

Comment:
>_uriPrefix [](http://example.com/codeflow?start=32&length=10)

can we remove and use httpClient?

Reply:
I don't think so
This one is only used for backward compatibility